blogposts/announcing-sourcegraph-3.14.md

```diff
@@ -0,0 +1,260 @@
+---
+title: "Sourcegraph 3.14: Improved permissions syncing, forks and archived repositories excluded by default, and docker-compose deployment"
+author: Christina Forney
+publishDate: 2020-03-20T10:00-07:00
+tags: [
+  blog
+]
+slug: sourcegraph-3.14
+heroImage: https://about.sourcegraph.com/blog/3.14-release-blog-img.jpg
+published: true
+---
+
+testing 123
+
+Sourcegraph is the standard developer platform for code search and navigation at many of the largest and most advanced technology companies. With Sourcegraph, every company has access to the same kind of tools that Google and Facebook developers use every day.
+
+<div style="padding-left: 2rem">
+
+[**🔑 Permissions syncing improvements for GitLab and Bitbucket Server**](#permissions-syncing-improvements-for-gitlab-and-bitbucket-server)<br />
+
+[**🍴 Forks and archived repositories are excluded from search results by default**](#forks-and-archived-repositories-are-excluded-from-search-results-by-default)<br />
+
+[**🐳 Deploy and scale Sourcegraph with docker-compose**](#deploy-and-scale-sourcegraph-with-docker-compose)<br />
+
+[**📣 Improved campaign creation, management, and syncing performance**](#improved-campaign-creation-management-and-syncing-performance)<br />
+
+[**⚠️ Required migration for Kubernetes deployments**](#required-migration-for-kubernetes-deployments)<br />
+
+[**📈 Improved debuggability of Kubernetes deployments**](#improved-debuggability-of-kubernetes-deployments)<br />
+
+[**🚨 Monitoring and alerting added to the Sourcegraph instrumentation**](#monitoring-and-alerting-added-to-the-sourcegraph-instrumentation)<br />
+
+[**🧮 Estimate required resources for your Sourcegraph instance**](#estimate-required-resources-for-your-sourcegraph-instance)<br />
+
+[**🙅‍♀️ Repository management now can exclude GitHub forks/archives**](#repository-management-now-can-exclude-github-forksarchives)<br />
```

Comment:
This phrasing is odd — not sure how to make it better, though 🤔 . Perhaps "Option to exclude all forks/archives from syncing to Sourcegraph for GitHub"?

Reply:
I was having trouble with that too - and making sure it sounded different enough from excluding forks/achives by default from searhc results. This does the trick I think! Thanks!